### PR TITLE
Fixed upload artifacts to release and set iptables rules

### PR DESCRIPTION
**Pull request by soffokl**

### CODE REVIEW

File: nat/service_iptables.go [hunk 1]
```diff
@@ -34,17 +34,9 @@ func (service *serviceIPTables) Add(rule RuleForwarding) {
 }
 
 func (service *serviceIPTables) Start() error {
-	err := service.ipForward.Enable()
-	if err != nil {
```
Comment (Waldz) [inline nat/service_iptables.go]: Maybe we should skipp this error here, in high level logic.
But lets have `ipForward.Enable` not swelling errors

Comment (soffokl) [reply nat/service_iptables.go]: Reverted error returning

File: nat/service_pfctl.go [hunk 2]
```diff
@@ -38,17 +38,9 @@ func (service *servicePFCtl) Add(rule RuleForwarding) {
 }
 
 func (service *servicePFCtl) Start() error {
-	err := service.ipForward.Enable()
-	if err != nil {
-		return err
-	}
-
+	service.ipForward.Enable()
```
Comment (Waldz) [inline nat/service_pfctl.go]: Warning logging whould be exactly here in high logic.
So that everybody understands why we swallow errors

Comment (soffokl) [reply nat/service_pfctl.go]: We have logging in the `Enable`, do you want to duplicate it or move it to `Start` only?

Comment (soffokl) [reply nat/service_pfctl.go]: Added extra log line.

File: nat/service_pfctl.go [hunk 3]
```diff
@@ -40,15 +40,11 @@ func (service *servicePFCtl) Add(rule RuleForwarding) {
 func (service *servicePFCtl) Start() error {
 	err := service.ipForward.Enable()
 	if err != nil {
-		return err
+		log.Warn(natLogPrefix, "Failed to enable IP forwarding: ", err)
```
Comment (Waldz) [inline nat/service_pfctl.go]: Now it's double loging, is not it?

Comment (soffokl) [reply nat/service_pfctl.go]: yeah, it prints messages from both functions.

Comment (soffokl) [reply nat/service_pfctl.go]: I think we can leave it this way for the release.

**Pull request merged**